aimage/aimage.py

```diff
@@ -321,13 +321,8 @@ async def _can_run_command(self, ctx: commands.Context, command_name: str) -> bo
         return can
 
     async def get_api_instance(self, ctx: Union[commands.Context, discord.Interaction]):
-        api_type = await self.config.guild(ctx.guild).api_type()
-        if api_type == API_Type.AUTOMATIC1111.value:
-            from aimage.apis.a1111 import A1111
-            instance = A1111(self, ctx)
-        elif api_type == API_Type.AIHorde.value:
-            from aimage.apis.aihorde import AIHorde
-            instance = AIHorde(self, ctx)
+        from aimage.apis.a1111 import A1111
```

Comment:
`aimage.apis.a1111` was removed, and this function is rendered useless.

`_update_autocomplete_cache()` won't work anymore because `get_api_instance` doesn't work now. Please fix.